docs/_source/docs/stack_group_config.rst

```diff
@@ -92,6 +93,18 @@ This will impact :ref:`Templating` of stacks by modifying the behavior of jinja.
       trim_blocks: True
       newline_sequence: \n
 
+http_template_handler
+~~~~~~~~~~~~~~~~~~~~~
+
+Optios passed to the `http template handler`_.
+  * retries - The number of retry attempts (default is 5)
+  * timeout - The timeout for the session in seconds (default is 5)
```

Comment:
I'm thinking that we might want to clarify these a little more so they don't accidentally conflict with other sceptre configs since these are so common of names. 

What about:
* http_handler_retries
* http_handler_timeout
?

Reply:
Never mind! I just saw that these are really sub-keys.